src/rtcModule/webrtc.cpp

```diff
@@ -2819,16 +2846,26 @@ void Session::veryfySdpOfferSendAnswer()
         if (wptr.deleted() || (mState > Session::kStateInProgress))
             return;
 
+        uint8_t opcode;
+        if (mState < kStateInProgress)
+        {
+            mTsSdpHandshakeCompleted = time(nullptr);
+            setState(kStateConnecting);
+            opcode = RTCMD_SDP_ANSWER;
+        }
+        else
```

Comment:
Should we handle the `mState >= kStateTerminating` here?? Not sure if it's a valid state at this codepath.

Reply:
It's checked few lines above